### DISCUSSION TITLE
How can I change my default file manager on LXQt 0.14.1?

### DISCUSSION THREAD

**Discussion by Yekutiel** [lxqt/lxqt#2197] [Q&A] [answered]
I am running LXQt 0.14.1 on Lubuntu 20.04 LTS.

I want my default file manager to be **SpaceFM**.  Currently my default file manager is **PCManFM**.

I read [Chapter 3.2.17 Alternatives Configurator](https://manual.lubuntu.me/stable/3/3.2/3.2.17/alternative_configurator.html) but it was "as clear as mud" to me (In other words, it didn't make sense to me).

**Comment by tsujan** [comment, marked as answer]
If you don't see `File Associations → Default Applications → File Manager`, your LXQt installation is too old. Use a distro with an up-to-date version.

**Reply by Yekutiel** [reply]
Thanks for letting me know. I appreciate that.

As I indicated, I am using Lubuntu 20.04 LTS currently.

However a newer version, Lubuntu 22.04 LTS (that 22.04 vs 20.04)  is due to be released in April 2022. I expect it will have have a newer version of LXQt. I am planning to upgrade to  Lubuntu 22.04 LTS in about 2 months or so. I'll wait until then.
